File: 02_Patterns/Pattern_13/Pattern_13.py
class Solution:
    def printTriangle(self, N):

        # num variable for print
        num = 1
        # outer loop for number of rows/lines
        for i in range(1, N+1):

            # inner loop for columns
            for j in range(1, i+1):
                print(num, end=" ")

                # increment num after print
                num += 1

            print()


# {
# Driver Code Starts
# Initial Template for Python 3
if __name__ == '__main__':
    t = int(input())
    for _ in range(t):
        N = int(input())
        ob = Solution()
        ob.printTriangle(N)
# } Driver Code Ends

File: 02_Patterns/Pattern_13/Pattern_13.js
function numberTriangle(n) {
  let currentNumber = 1;

  // Loop for number of lines/rows
  for (let i = 1; i <= n; i++) {
    let row = '';

    // loop for number of columns to print numbers for each line
    for (let j = 1; j <= i; j++) {
      // Append the current number to the row
      row += currentNumber + ' ';
      // Increment the current number for the next iteration
      currentNumber++;
    }

    // Print the row to the console
    console.log(row);
  }
}

// Call the function
// numberTriangle(5);

// Export the function for testing
module.exports = numberTriangle;
